cli/sawtooth_cli/network_command/peers.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import os
+import json
+import shlex
+import subprocess
+
+from sawtooth_cli.exceptions import CliException
+
+
+DOT_FILE = 'peers.dot'
+PNG_FILE = 'peers.png'
+
+
+def add_peers_parser(subparsers, parent_parser):
+    parser = subparsers.add_parser(
+        'peers',
+        help='TODO',
+        description='TODO',
+    )
+
+    peers_parsers = parser.add_subparsers(
+        title='subcommands',
+        dest='peers_command')
+    peers_parsers.required = True
+
+    _add_list_parser(peers_parsers)
+    _add_graph_parser(peers_parsers)
+
+
+def _add_list_parser(parser):
+    list_parser = parser.add_parser(
+        'list',
+        help='List peers for validators with given URLs',
+        description=(
+            'List peers for validators with given URLs separated by commas.'))
+
+    list_parser.add_argument(
+        'urls',
+        nargs=1,
+        help='the comma-separated URLs of the validators to be queried',
+        type=str)
```

Comment:
You should use `cli/sawtooth_cli/network_command/parent_parsers.py` for this as it supports authenticated access with the rest apis. It also has a function to create the rest clients for all the peers.

Reply:
It also supports passing the urls as a space separated list, which is how compare-chains works, and we should provide a consistent way to pass urls.